app/views/assignments/_form.html.erb

```diff
@@ -212,13 +212,23 @@
       <%= f.fields_for :assignment_files do |assignment_file_form| %>
         <%= render partial: 'assignment_file',
                    locals: { assignment_file: assignment_file_form.object,
-                             form: f } %>
+                             form: f } %> 
       <% end %>
     </div>
-
      <p id='assignment_file_add_link'>
         <%= add_assignment_file_link t('assignment.add_required_file'), f %>
      </p>
+     </ br>
+     <%= f.check_box :is_required,
+       checked: @assignment.is_required,
```

Comment:
The alignment in this and the f.label tags is off. The arguments should all be vertically aligned, so here `checked` should be aligned with `:is_required`, etc.